tests/AuthAPITestCase.py

```diff
@@ -1,7 +1,8 @@
 from rest_framework.test import APITestCase, APIClient
 from users.models import User
 from org.models import *
-
+from org.custom_model_field import PermissionSet as Permissions
+import uuid
 
```

Comment:
1. I suppose you misunderstood the suggestions about `tests/AuthAPITestCase.py `.
1. The purpose is not to alter this file but to use it ( instead of duplicating the `create_auth_client()` in every test.
1. Please revoke the changes you made to this file :smiley: